wikimedia/wikipedia-ios
# Diffs - Show edit history when menu item tapped

**Phabricator:** https://phabricator.wikimedia.org/T335579 (partially)

### Notes
Show the full edit history view when tapped from the toolbar menu item. I copied the `pageURL` construction methodology from the recent lead image URL fetching work here – if there's a more accepted way of constructing this URL I'd be happy to change this.

### Test Steps
1. Open a pair of diffs (either by way of page history, or from mobile web to app)
2. Tap more > article edit history in the toolbar.
3. Confirm the full article page history opens.
4. Confirm paging works correctly when scrolling this list of diffs.
